README.md

```diff
@@ -1 +1,38 @@
-# webpack-exercise
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] The pull request should be more descriptive, you should explain the things that you've added in this feature branch